src/components/Farmhand/Farmhand.js

```diff
@@ -214,6 +215,7 @@ const applyPriceEvents = (valueAdjustments, priceCrashes, priceSurges) => {
  * @property {number} experience
  * @property {string} farmName
  * @property {(?farmhand.plotContent)[][]} field
+ * @property {(?farmhand.plotContent)[][]} forest
```

Comment:
that's a really interesting, and pretty clever, approach. i was imagining the tree plots and the foragable plots would just be two separate lists, but we would render them in such a way that they appeared intertwined with each other. i'm not really feeling like one approach is any better than the other here, though it feels simpler to me to manage 2 separate lists and just deal with rendering positioning for making it look how we want.

Reply:
We could choose to keep the data simpler (by not having `null` in the middle of the sub-nested array) and use logic to enforce a valid structure, but to me that seems a bit more complex to manage. Using a complex data structure would also enable the type checker to enforce type safety. Either way, I think it'll take some experimentation to see what works best here.

Reply:
i think the key difference between our thoughts of the forest is you're imagining a pretty dense forest grid where potentially every space has something in it, like this: (where `F` = foragable and `T` = tree)

```
| F | F | F | F | F | F | F | F | F |
| F | T | F | F | T | F | F | T | F |
| F | F | F | F | F | F | F | F | F |
```

and i'm thinking of something that's a bit more spacious, and uneven, like this: (`X` = blank space)

```
| F | X | X | F | F | X | X | X | F |
| X | T | X | F | X | F | X | T | X |
| X | F | X | X | F | X | F | X | F |
```

so i'm seeing it as either a grid where certain cells never have anything (maybe decor), some are valid foragable spots, and some are valid tree spots. or.. i'm  just a simple set of two lists (`treePlots` and `foragablePlots`) which can each hold a specific typed object,  and the rendering ends up positioning them how we want rather than enforcing "only __ can go in cell #" for each cell.

the sub-nested array you've proposed seems a little bit harder to manage to me because if you want to get at the foragable cell (3, 3) you would have to know it's actually a sub-cell of (0, 0), i think?

one more thing i want to mention is that although the trees have been drawn in 36x72 sprites, i was imagining the cells be 36x36 so that the trees can overlap 2 cells visually, which i think potentially makes things more interesting to look at, but it does mean there would be no point in spawning a foragable in the cell just above a tree cell because it wouldn't be noticeable. we could definitely make the cells 36x72 and avoid this, but we also might run out of vertical space a lot faster if we do that. 

i still think all the ways we're talking about this data are valid, but hopefully that little illustration is helpful to compare how i think we're each thinking about this.

Reply:
I see what you're saying. Given the design you're describing, I think it makes sense to go with a simpler data structure (just a nested list) to represent the entirety of the forest. Perhaps something like `(farmhand.plantedTree | farmhand.forestForageable | null)[][]`? I think we'll want to have some game rules around how closely things can be placed (either by the player planting saplings or random foragable spawn locations). That's something that can be figured out in a later iteration.

Good thinking!